lib/common.sh

```diff
@@ -41,8 +41,20 @@ INT_IF=${INT_IF:-}
 ROOT_DISK_NAME=${ROOT_DISK_NAME-"/dev/sda"}
 # Hostname format
 NODE_HOSTNAME_FORMAT=${NODE_HOSTNAME_FORMAT:-"node-%d"}
+
 # Check OS type and version
-OS=$(awk -F= '/^ID=/ { print $2 }' /etc/os-release | tr -d '"')
+# shellcheck disable=SC1091
+source /etc/os-release
+export DISTRO="${ID}${VERSION_ID%.*}"
+export OS="${ID}"
+export OS_VERSION_ID=$VERSION_ID
+
+if [[ $DISTRO != "centos8" && $DISTRO != "rhel8" && $DISTRO != "ubuntu20" && $DISTRO != "ubuntu18"  ]]; then
```

Comment:
since we're here, I think would be much cleaner if we moved the supported distributions to a variable, like:
SUPPORTED_DISTROS="centos8|rhel8|ubuntu20|ubuntu18"
and then let the test evaluate if DISTRO is not in the list using the =~ operator

Reply:
Great, now I got what Steven was saying :). Thanks @elfosardo .